local-antora-playbook.yml

```
@@ -17,7 +17,7 @@ content:
   - url: https://github.com/redpanda-data/docs
     branches: [v/*, api, shared, site-search,'!v-end-of-life/*']
   - url: https://github.com/redpanda-data/cloud-docs
-    branches: 'main'
+    branches: 'DOC-1216-Document-how-to-pass-in-secrets-when-configuring-iceberg'
```

```suggestion
    branches: 'main'
```
add commit before merge